### PR TITLE
Minor bugfixes from testing

### PR DESCRIPTION
**Pull request by kushalbakshi**

### CODE REVIEW

File: element_calcium_imaging/imaging_no_curation.py [hunk 1]
```diff
@@ -517,7 +517,7 @@ def make(self, key):
 
                         plane_processing_tasks = []
                         for plane_idx in PVmeta.meta["plane_indices"]:
-                            pln_output_dir = output_dir / f"pln{plane_idx}_chn{channel}"
+                            pln_output_dir = pathlib.Path(output_dir + f"/pln{plane_idx}_chn{channel}")
```
Comment (ttngu207) [inline element_calcium_imaging/imaging_no_curation.py]: Why do we need this?
`output_dir` should already be a `Path` object at this step in the code

Comment (kushalbakshi) [reply element_calcium_imaging/imaging_no_curation.py]: I get this error if I use the `/` there without `pathlib.Path()`: `TypeError: unsupported operand type(s) for /: 'str' and 'str'`

**Review by ttngu207 — APPROVED** [review]

File: element_calcium_imaging/imaging_no_curation.py [hunk 2]
```diff
@@ -517,7 +523,9 @@ def make(self, key):
 
                         plane_processing_tasks = []
                         for plane_idx in PVmeta.meta["plane_indices"]:
-                            pln_output_dir = output_dir / f"pln{plane_idx}_chn{channel}"
+                            pln_output_dir = pathlib.Path(
+                                output_dir + f"/pln{plane_idx}_chn{channel}"
+                            )
```
Comment (ttngu207) [inline element_calcium_imaging/imaging_no_curation.py]:
```suggestion
                            pln_output_dir = pathlib.Path(output_dir) / f"pln{plane_idx}_chn{channel}"
                            
```

Comment (ttngu207) [reply element_calcium_imaging/imaging_no_curation.py]: @kushalbakshi just one small suggestion

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**